test/file.jl

```diff
@@ -50,6 +50,20 @@ end
 
 using Random
 
+@testset "that temp names are actually unique" begin
+    temps = [tempname(cleanup=false) for _ = 1:1000]
```

Comment:
This has an expected failure rate on our CI of 0.001% (`1 - e^(-1000^2 / 62^6 / 2)`) which means 1 in 100k CI runs (or about 1 in 10k commits) would be expected to fail this test (over the current lifetime of this project, that would mean there'd have been about 2-3 failures)

Reply:
I don't follow. The filenames that are generated in this PR are 128 bit UUIDs, which, as far as I can tell, will lead to a way, way, way lower collision rate than what you compute here.

Reply:
I think the point that Jameson is making is that this test is unnecessarily strict, which I agree (I only used 1000 to test the PR). Something like 100, is more than sufficient.

Reply:
At least this does not use Julia's RNG, which gets reset on every `@testset`.